src/classes/OPP_OpportunityContactRoles_TEST.cls

```diff
@@ -34,31 +34,31 @@
 */
 @isTest
 private class OPP_OpportunityContactRoles_TEST {
+
+    /*********************************************************************************************************
+    * @description if you only want to run one test in this class, fill in its name here.
+    * if you want to run all tests, then use '*'
+    */
+    private static string strTestOnly = 'avoidDuplicateCRs';
```

Comment:
Blarg, I need some sort of automation to remind me of this :)

On Mon, Dec 28, 2015 at 4:46 PM, David Habib <notifications@github.com>
wrote:

> In src/classes/OPP_OpportunityContactRoles_TEST.cls
> <https://github.com/SalesforceFoundation/Cumulus/pull/1863#discussion_r48515676>
> :
>
> > @@ -34,31 +34,31 @@
> >  */
> >  @isTest
> >  private class OPP_OpportunityContactRoles_TEST {
> > +
> > +    /*********************************************************************************************************
> > +    * @description if you only want to run one test in this class, fill in its name here.
> > +    * if you want to run all tests, then use '*'
> > +    */
> > +    private static string strTestOnly = 'avoidDuplicateCRs';
>
> think you want this back to *!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SalesforceFoundation/Cumulus/pull/1863/files#r48515676>
> .
>